### ISSUE TITLE
About the training log of each loss function

### ISSUE THREAD

**Issue by whyandbecause** [Ephemeral182/UDR-S2Former_deraining#11]
Thank you for your great work. I would like to ask if you can provide the training log as a reference.  I would like to find whether the changes of each training loss in your training version are consistent with mine. Thank you！

**Comment by Ephemeral182** [comment]
Hello, I seem to have not found my training logs. If possible, could you send me your training logs? I'd be happy to help review them and check for any potential issues.
